.buildconfig.yml

```diff
@@ -47,6 +47,10 @@ projects:
     path: components/concept/menu
     description: 'An abstract definition of a browser menu component.'
     publish: true
+  concept-pocket:
+    path: components/concept/pocket
+    description: 'An abstract definition of a component for displaying Pocket recommendations'
+    publish: true
```

Comment:
Are those definitions needed anywhere else, where we do not want to depend on the feature? Otherwise I am not sure we need a `concept`? The other reason, supporting multiple implementations, doesn't apply here either. :)

Reply:
No. Not used anywhere else now.
My plan is to have all the functionality and also the ui code in AC with Fenix (maybe others also) just having to use it as a `LifecycleAwareFeature`. Kind of a Pocket SDK but with most things interchangeable - like the data provider, maybe some other app doesn't want to use our `service-pocket` but wants the functionality (incl the UI). 
For this I though I'd need to follow this basic structure:
- any functionality needs the basic contracts in a `concept`
- then the implementation would be a `component-...`, 
- integrated with a `feature-...`
- with the above two possibly using any `service` / `lib` / `support` dependencies, as needed.

@pocmo Can you advise if this sounds ok and what the best structure would be?

Reply:
Fenix integration looks good! I was also wondering about the concept component though. To me this would only be needed if it contained generic contracts for recommended articles, where pocket is one implementation, and we anticipated others. I don't think that's the case though? The split of feature/service makes more sense to me as the service could be re-usable in other apps without the UI integration. So, the Fenix side of this change looks good to me.